test/gui/shared/scripts/helpers/SetupClientHelper.py

```diff
@@ -0,0 +1,11 @@
+class SetupClientHelper():
+    def __int__(self):
+        pass
+    
+    def substituteInLineCodes(self, context, value):
```

Comment:
can we just use this as a function without the class, it would be much easier to use since we don't have to create new object every time.

Reply:
Or, maybe we can make that method static using `@staticmethod` decorator

Reply:
making the fn static method might be a nice solution